manticore/native/models.py

```diff
@@ -140,3 +142,123 @@ def strlen(state, s):
             ret = ITEBV(cpu.address_bit_size, byt == 0, offset, ret)
 
     return ret
+
+
+def is_NULL(byte, constrs) -> bool:
```

Comment:
You might consider renaming this to something like `is_definitely_NULL`, which might be a bit clearer as to what it's testing.